src/components/networked-video-source-green-screen.js

```diff
@@ -0,0 +1,164 @@
+/* global AFRAME, NAF, THREE */
+var naf = require('../NafIndex');
+
+AFRAME.registerComponent('networked-video-source-green-screen', {
+
+  schema: {
+    streamName: { default: 'video' },
+    GreenThresholdIn : {default: 0.02},
+    event: {type: 'string', default: ''}
+  },
+
+  dependencies: ['material'],
+
+  update: function () {
+
+    var data = this.data;  // Component property values.
+    var el = this.el;  // Reference to the component's entity.
+
+    if (data.event) {
+      // This will log the `message` when the entity emits the `event`.
+      el.addEventListener(data.event, function () {
+      });
```

Comment:
dead code?

Reply:
oops. it was from debugging. Yes delete it.